helm/values.schema.json

```diff
@@ -0,0 +1,1713 @@
+                              "my-notary-auth-creds"
+                            ]
+                          }
+                        },
+                        "required": [
+                          "secret"
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "required": [
+                  "host",
+                  "trust_roots"
+                ]
+              }
+            },
+            {
+              "$id": "#/properties/validators/items/anyOf/1",
+              "type": "object",
+              "title": "The second anyOf schema",
+              "description": "A notary V2 validator instance. This validator is not functional at this point of time and just reserved for future use.",
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "notaryv2"
+                  }
+                }
+              },
+              "then": {}
+            },
+            {
+              "$id": "#/properties/validators/items/anyOf/2",
+              "type": "object",
+              "title": "The third anyOf schema",
+              "description": "A cosign validator instance.",
+              "examples": [
+                {
+                  "trust_roots": [
+                    {
+                      "name": "mykey",
+                      "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni\nqiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==\n-----END PUBLIC KEY-----\n"
+                    }
+                  ]
+                }
+              ],
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "cosign"
+                  }
+                }
+              },
+              "then": {
+                "properties": {
+                  "trust_roots": {
+                    "$id": "#/properties/validators/items/anyOf/2/properties/trust_roots",
+                    "type": "array",
+                    "title": "The trust_roots schema for cosign validator",
+                    "description": "The trust root of the cosign validator. Currently, only public keys are supported as trust roots, but there may come other kinds of trust roots (e.g. mail addresses with OIDC flow).",
+                    "examples": [
+                      [
+                        {
+                          "name": "mykey",
+                          "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni\nqiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==\n-----END PUBLIC KEY-----\n"
+                        }
+                      ]
+                    ],
+                    "items": {
+                      "$id": "#/properties/validators/items/anyOf/2/properties/trust_roots",
+                      "anyOf": [
```

Comment:
Does this anyOf have only one entry on purpose for easier extendability?

Reply:
Yes, I think I probably did it like this because we once said we are going to offer different kinds of trust roots in the future. Can also remove this, if you think it's unnecessary or confusing right now - do you? :)

Reply:
No, I think its fine if you want to define it this way for easy extendability. Just making sure its conscious